node/src/service.rs

```diff
@@ -207,6 +207,13 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
         })?;
 
     if config.offchain_worker.enabled {
+        sp_keystore::SyncCryptoStore::sr25519_generate_new(
```

Comment:
This is for testing purpose only so that the whole thing works when run locally with --dev.
Will be removed before merge.